actionpack/lib/abstract_controller/translation.rb

```diff
@@ -20,7 +26,9 @@ def translate(key, **options)
         options[:default] = defaults.flatten
         key = "#{path}.#{action_name}#{key}"
       end
-      I18n.translate(key, **options)
+
+      i18n_raise = options.fetch(:raise, ActionController::Base.raise_on_missing_translations)
```

Comment:
Referencing `ActionController::Base` here will cause it to be loaded for API-only applications that include this module manually, which is documented here:

https://guides.rubyonrails.org/api_app.html#adding-other-modules

Maybe the configuration option should be defined directly on this module?